tests/scripts/makeTestImages.sh

```diff
@@ -1,37 +1,50 @@
-#!/bin/bash -e
+#!/bin/bash
+set -ex
+
+# readable bash output
+export PS4='+${BASH_SOURCE}:${LINENO}: ${FUNCNAME[0]:+${FUNCNAME[0]}(): }'
 
 scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 # shellcheck disable=SC1090
 source "${scriptdir}/../../build/common.sh"
 
+# list of images to build
+imagesToBuild=(ceph cockroachdb cassandra edgefs nfs yugabytedb)
```

Comment:
I know I suggested to add edgefs to the list, but now I see that it wasn't previously found anywhere in this file. For consistency, let's not add it to the list, and separately investigate the implications of it not being in this list.